src/status_im2/contexts/quo_preview/drawers/permission_drawers.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.quo-preview.drawers.permission-drawers
+  (:require [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im.react-native.resources :as resources]))
+
+(defn example-1
+  []
+  [:<>
+   [quo/text {:style {:margin-right 4}} "Hold"]
+   [quo/permission-tag
+    {:size             24
+     :locked           false
+     :tokens           [{:id    1
+                         :group [{:id 1 :token-icon (resources/get-image :status-logo)}
+                                 {:id 2 :token-icon (resources/get-image :status-logo)}
+                                 {:id 3 :token-icon (resources/get-image :status-logo)}
+                                 {:id 4 :token-icon (resources/get-image :status-logo)}
+                                 {:id 5 :token-icon (resources/get-image :status-logo)}]}]
+     :background-color (colors/theme-colors
+                        colors/neutral-10
+                        colors/neutral-80)}]
+   [quo/text
+    {:style {:margin-left  4
+             :margin-right 4}} "Or"]
+   [quo/permission-tag
+    {:size             24
+     :locked           false
+     :tokens           [{:id    1
+                         :group [{:id 1 :token-icon (resources/get-image :status-logo)}
+                                 {:id 2 :token-icon (resources/get-image :status-logo)}
+                                 {:id 3 :token-icon (resources/get-image :status-logo)}
+                                 {:id 4 :token-icon (resources/get-image :status-logo)}
+                                 {:id 5 :token-icon (resources/get-image :status-logo)}]}]
+     :background-color (colors/theme-colors
+                        colors/neutral-10
+                        colors/neutral-80)}]
+   [quo/text {:style {:margin-left 4}} "To post"]])
+
+(defn example-2
+  []
+  [:<>
+   [quo/text {:style {:margin-right 4}} "Hold"]
+   [quo/token-tag
+    {:size          :small
+     :token-img-src (js/require "../resources/images/tokens/mainnet/ETH.png")} "ETH"]
+   [quo/text {:style {:margin-left 4}} "To post"]])
+
+(defn example-3
+  []
+  [:<>
+   [quo/icon :i/communities {:color (colors/theme-colors colors/neutral-100 colors/white)}]
+   [quo/text {:style {:margin-right 4}} "Join community to post"]])
+
+(defn cool-preview
+  []
+  (fn []
+    [:<>
+     [rn/view {:margin-top 60}
+      [quo/permission-context [example-1] #(js/alert "drawer pressed")]]
+     [rn/view {:margin-top 60}
+      [quo/permission-context [example-2] #(js/alert "drawer pressed")]]
+     [rn/view {:margin-top 60}
+      [quo/permission-context [example-3] #(js/alert "drawer pressed")]]]))
+(defn preview-permission-drawers
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/neutral-20 colors/neutral-100)
```

Comment:
Less important in preview files perhaps, but we're trying to put the style map inside the `:style` key.